src/main/java/org/testeditor/fixture/host/HostDriverFixture.java

```
@@ -110,20 +110,24 @@ public HostDriverFixture(Connection connection) {
      */
     @FixtureMethod
     public boolean connect(String s3270Path, String hostname, int port, int offsetRow, int offsetColumn) throws FixtureException {
-        logger.info("Host-Fixture connecting ...");
+        logger.debug("Host-Fixture connecting ...");
         this.offset = new Offset(-offsetRow, -offsetColumn);
+        logger.trace("Offset to original s3270 screen X-Position = {} ; Y-Position = {}.", offsetColumn, offsetRow); 
         TerminalType type = TerminalType.TYPE_3279;
         CharacterSet charSet = CharacterSet.CHAR_GERMAN_EURO;
+        logger.trace("Used parameters for connection:\n Terminal Type : {} ;\n Character Set : {} ;\n Path to s3270 application : {} ;"
+                + "\n Connected Hostname : {} ;\n Connected Port : {}" , type , charSet, s3270Path, hostname, port);
+        
         timer.startTimer();
         connection.connect(s3270Path, hostname, port, type, terminalMode, charSet, offset);
         if (connection.isConnected()) {
             waitUntilScreenIsFormatted(MAX_TIME_TO_WAIT);
             timer.stopTimer();
-            logger.debug("Elapsed time in millis after connect: {}", timer.getElapsedTime());
-            logger.info("successfully connected to host='{}', port='{}'", hostname, port);
+            logger.trace("Elapsed time in millis after connect: {}", timer.getElapsedTime());
+            logger.debug("successfully connected to host='{}', port='{}'", hostname, port);
             return true;
         } else {
-            logger.info("The connection to host '" + hostname + "' on port '" + port + "' could not be established.");
+            logger.error("The connection to host '" + hostname + "' on port '" + port + "' could not be established.");
```

throw FixtureException instead of returning false